passcodeview/src/main/java/online/devliving/passcodeview/PasscodeView.java

```diff
@@ -221,7 +244,7 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             public void afterTextChanged(Editable s) {
                 int length = s.length();
                 updateChilViewSelectionStates(length, mEditText.hasFocus());
-
+                mPasscodeEntryListener.onPasscodeDigitEntered();
```

Comment:
@dryaz shouldn't we check if `mPasscodeEntryListener` is null here?

Reply:
You're right, yes, thx, will update after my vacation in ~1 week